### DISCUSSION TITLE
Rocky linux is supported or not?

### DISCUSSION THREAD

**Discussion by mykishhu** [aquasecurity/trivy#5368] [Q&A]
### Question

I didn't find rocky linux in supported OS list by trivy.
But in Rocky linux page it says we can use Trivy.
Some confusion here.

AIX is also not listed in supported OS. does Trivy support AIX?

### Target

Virtual Machine Image

### Scanner

Vulnerability

### Output Format

None

### Mode

None

### Operating System

Rocky Linux or AIX

**Comment by knqyf263** [comment]
Supported
https://aquasecurity.github.io/trivy/v0.45/docs/coverage/os/rocky/
